### PR TITLE
Imaging viewer tab

### PR DESCRIPTION
**Pull request by john-squire**

### CODE REVIEW

**Comment by avaidyam** [comment]
Can you include screenshots?

File: public/img/4206942069/images.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  ""
```
Comment (avaidyam) [inline public/img/4206942069/images.json:2]: This is not valid JSON. Please delete it!

Comment (john-squire) [reply public/img/4206942069/images.json:2]: It's a placeholder, but I'll delete it

**Comment by john-squire** [comment]
image
Screenshot of radiology screen

**Pull request merged**